src/users/user.php

```diff
@@ -112,11 +105,12 @@ public function post()
 			}
 			else
 			{
-				if ($fidentifier)
+				// Forced user creation
+				if ($fIdentifier)
 				{
 					$user = new JUser;
```

Comment:
If id is provided, and is exist then user should get updated
@manojLondhe @coolbung